### DISCUSSION TITLE
Filter "today and unread"

### DISCUSSION THREAD

**Discussion by spenat28** [ViennaRSS/vienna-rss#1342] [Ideas]
Filtering is not very user friendly.
I would suggest at least adding some options like "Today and unread" due to the fact, filter is shared among all subscriptions.

Better solution would be to add some buttons to fast filter unread/all/today ... instead you have to click on filter option, move your mouse to select one, click again.

**Comment by josh64x2** [comment]
@spenat28 are the default smart folders not accomplishing the same task?

![image](https://user-images.githubusercontent.com/799345/97771390-8117e700-1b90-11eb-89ef-7398d8e247b1.png)

**Comment by barijaona** [comment]
Smart folders aggregate all feeds. I guess that @spenat28 wants to navigate between feeds.
But what would be the benefit of "Today and unread" over "Unread"? What is the strategy regarding backlog articles?
